examples/kmeans/GetIntelGpuResources.sh

```diff
@@ -17,4 +17,4 @@
 #ADDRS="0","1","2","3","4","5","6","7"
 #echo {\"name\": \"gpu\", \"addresses\":[\"$ADDRS\"]}
 
-echo {\"name\": \"gpu\", \"addresses\":[\"0\",\"1\",\"2\",\"3\"]}
+echo {\"name\": \"gpu\", \"addresses\":[\"0\"]}
```

Comment:
It's better to keep at least 2 GPUs as example. pls revise below as well. And could we move this file to a common place since it's common for all GPU examples.